multiaccounts/accounts/database.go

```diff
@@ -70,19 +75,19 @@ func (db Database) Close() error {
 	return db.db.Close()
 }
 
-func (db *Database) GetAccounts() ([]Account, error) {
-	rows, err := db.db.Query("SELECT address, wallet, chat, type, storage, pubkey, path, name, emoji, color, hidden, derived_from FROM accounts ORDER BY created_at")
+func (db *Database) GetAccounts() ([]*Account, error) {
+	rows, err := db.db.Query("SELECT address, wallet, chat, type, storage, pubkey, path, name, emoji, color, hidden, derived_from, clock FROM accounts ORDER BY created_at")
 	if err != nil {
 		return nil, err
 	}
 	defer rows.Close()
-	accounts := []Account{}
+	accounts := []*Account{}
 	pubkey := []byte{}
 	for rows.Next() {
-		acc := Account{}
+		acc := &Account{}
 		err := rows.Scan(
 			&acc.Address, &acc.Wallet, &acc.Chat, &acc.Type, &acc.Storage,
-			&pubkey, &acc.Path, &acc.Name, &acc.Emoji, &acc.Color, &acc.Hidden, &acc.DerivedFrom)
+			&pubkey, &acc.Path, &acc.Name, &acc.Emoji, &acc.Color, &acc.Hidden, &acc.DerivedFrom, &acc.Clock)
```

Comment:
We need to be careful on upgrade, since the value will be Null on existing accounts, generally either you set it in the migration, or you use `sql.NullInt64`

Reply:
In the migration sql we specify `ADD COLUMN clock INT NOT NULL DEFAULT 0;`. So it should always be 0 on existing accounts. Not sure though.
I've tried changing the `Account.Clock` type to `sql.NullInt64` before noticing the `NOT NULL` constraint and then ran some tests in the `api` suite. Turned out that lines like `db.SaveAccounts([]*accounts.Account{{Address: address}})` begin to fail, because by default `sql.NullInt64` will try to insert a NULL value which is forbidden.

Reply:
@cammellos do you think `NOT NULL` constraint is enough?

Reply:
I can't remember what's the behavior for INT, maybe that works fine, it would have to be tested (create account with previous version, upgrade app, login).
Regarding nullInt:
Only when scanning you would use `sql.NullInt64` and then assign if not null to `Account.Clock` similarly to https://github.com/status-im/status-go/blob/997708bc82c68c842512700ee4a6ccdb04411f94/protocol/persistence.go#L276